app/api/items/navigationDataMapper.go

```diff
@@ -43,9 +44,16 @@ func getRawNavigationData(dataStore *database.DataStore, rootID int64, user *dat
 
 	commonAttributes := "items.ID, items.sType, items.bTransparentFolder, items.idItemUnlocked, items.idDefaultLanguage, fullAccess, partialAccess, grayedAccess"
 	itemQ := items.VisibleByID(user, rootID).Select(commonAttributes + ", NULL AS idItemParent, NULL AS idItemGrandparent, NULL AS iChildOrder, NULL AS bAccessRestricted")
+	if itemQ.Error() == database.ErrUserNotFound {
+		return nil, database.ErrUserNotFound
+	}
+	service.MustNotBeError(itemQ.Error())
 	childrenQ := items.VisibleChildrenOfID(user, rootID).Select(commonAttributes + ",	idItemParent, NULL AS idItemGrandparent, iChildOrder, bAccessRestricted")
+	service.MustNotBeError(childrenQ.Error())
```

Comment:
Don't you transform a forbidden into a 500 with that?  
I supposed it is tested 🙂

Reply:
Yes, it is tested. We return at line 48 if forbidden